application/ledger_query_service/src/ethereum/queries.rs

```diff
@@ -26,6 +27,89 @@ pub struct EthereumTransactionQuery {
     transaction_data_length: Option<usize>,
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize, Default)]
+pub struct EthereumTransactionLogQuery {
```

Comment:
:question: Why is there a need for a different `Query`? Can this not be part of `/transactions` or `/blocks`?

Reply:
This query only looks at transaction receipt logs and allows for skipping blocks which don't contain transactions emitting said logs, so the implementation is very different.